libraries/animation/src/AnimBlendDirectional.cpp

```diff
@@ -96,7 +96,9 @@ const AnimPoseVec& AnimBlendDirectional::evaluate(const AnimVariantMap& animVars
             }
         }
         _poses.resize(minSize);
-        blend4(minSize, &poseVecs[0][0], &poseVecs[1][0], &poseVecs[2][0], &poseVecs[3][0], &alphas[0], &_poses[0]);
+        if (minSize) {
```

Comment:
Since it is not a boolean of a pointer I would prefer to see an actual comparison. While your solution probably works, it does make it look like a `if something exists` while you actually want to see right away that it should not be 0 or maybe 0 or lower?
```suggestion
        if (minSize != 0) {
```
```suggestion
        if (minSize > 0) {
```